apps/store/src/components/TierSelector/TierSelector.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { useState } from 'react'
+import { CurrencyCode } from '@/services/apollo/generated'
+import { TierSelector, TierSelectorProps } from './TierSelector'
+
+export default {
+  title: 'Car Tier Selector',
+  component: TierSelector,
+} as ComponentMeta<typeof TierSelector>
+
+const MockedOffers: TierSelectorProps['offers'] = [
+  {
+    id: '0',
+    variant: { displayName: 'Trafikförsäkring', typeOfContract: 'Traffic' },
+    price: { amount: 399, currencyCode: CurrencyCode.Sek, __typename: 'Money' },
+  },
+  {
+    id: '1',
+    variant: { displayName: 'Halvförsäkring', typeOfContract: 'Halvförsäkring' },
+    price: { amount: 299, currencyCode: CurrencyCode.Sek, __typename: 'Money' },
+  },
+  {
+    id: '2',
+    variant: { displayName: 'Helförsäkring 🍒', typeOfContract: 'Driving' },
+    price: { amount: 999, currencyCode: CurrencyCode.Sek, __typename: 'Money' },
+  },
+]
+
+const Template: ComponentStory<typeof TierSelector> = (props) => {
+  const [selected, setSelected] = useState(props.selectedOfferId)
```

Comment:
```suggestion
  const [selected, setSelected] = useState(MockedOffers[0].id)
```

It's not possible to interact with the story at the moment.

<img width="297" alt="image" src="https://user-images.githubusercontent.com/19200662/207070341-ece3864e-79c5-467e-94a1-06ac7fe3e199.png">